lua/bufferline/buffer.lua

```diff
@@ -162,9 +168,9 @@ local buffer = {
     if hide.alternate or hide.current or hide.inactive or hide.visible then
       local shown = {}
 
-      hide = {hide.inactive, hide.visible, hide.alternate, hide.current}
```

Comment:
This was an instance of oversight (the one I mentioned earlier). In 4749af7dcfff4c4e8305bac4bd3734dc256d4d91 the ordering of `alternate` and `visible` was swapped, but I forgot to do that here. Now, we don't need to futz with ordering, we can just do `activity:lower()` to check if something is hidden.